app/src/main/java/org/fossasia/openevent/app/common/data/network/EventService.java

```diff
@@ -82,4 +83,7 @@
 
     @GET("events/{id}/general-statistics")
     Observable<EventStatistics> getEventStatistics(@Path("id") long id);
+
+    @GET("/v1/events/{id}/faqs?include=event&fields[event]=id&page[size]=0")
```

Comment:
Should be `events/{id}/faqs?include=event&fields[event]=id&page[size]=0`

Reply:
Oops! just corrected it.